app/controllers/businesses_controller.rb

```diff
@@ -33,9 +33,25 @@ def update
     redirect_to show_business_path(@business.slug)
   end
 
+  def destroy
+    business = Business.find(params[:id])
+    business.destroy
+    redirect_to :back
```

Comment:
should probably have flash messages to indicate that the business has been destroyed.